### DISCUSSION TITLE
how to set"Show all databases"?

### DISCUSSION THREAD

**Discussion by YoYogurt** [dbeaver/dbeaver#9276] [Q&A] [answered]
#### System information: 
- win10
- DBeaver7.1.1

#### Connection specification:
- Database name and version

#### Describe your question:
I created a database, but i cann't see it. And it tell me "it is already existed". How can I see the database I built？？？
and how to set"Show all databases"?

**Comment by uslss** [comment, marked as answer]
To see newly created db - Refresh connection in Connection's Context Menu
To set Show all database, e.g., for PostgreSQL check Show all database option in Connection Settings 
![image](https://user-images.githubusercontent.com/31996417/87664711-6864fc00-c76e-11ea-922e-36b732f52f83.png)

**Reply by MilenaValenze** [reply]
Just adding, because it took me a while to find:
To access this screen, just right-click on the connection and go to "Edit Connection (F4)"

**Comment by JoshElias** [comment]
Just wondering why this setting isn't enabled by default? If the user you're using for the connection has all the right permissions, including Super User, why would they not see all DBs by default?

**Comment by ghost** [comment]
This seems to have changed again. I had to change my `Connection View` to `Advanced`, then I was able to find the `Show all databases` option.

Although now that I've toggled the `Connection View` even if I set it back to `Simple` I'm still seeing the Show all databases option. So I'm not sure what state DBeaver is in out of the box.

I'm using DBeaver 25.0.4

**Reply by anthonymag** [reply]
This doesn't seem to bring the option back for me.

I'm on the Linux version, if it matters.

EDIT: this has moved. It's now in the main panel
Edit connection > Connection settings > Main > Check "show all databases"

**Comment by huaihuailj** [comment]
您好，我已收到您的邮件，我会尽快回复~
